### DISCUSSION TITLE
Alternate way for optional parameters in v6?

### DISCUSSION THREAD

**Discussion by iamPiyushNagpal** [remix-run/react-router#8345] [Q&A] [answered]
In v5, we could add trailing `?` to route for optional parameters, but as in v6, the support for the same has been dropped, so what's the alternate way of writing the following piece of code?

`<Route path="/cart/:id?" component={<CartPage />} />`

**Comment by brophdawg11** [comment, marked as answer]
Superseded by https://github.com/remix-run/react-router/issues/9546
